go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -518,7 +518,7 @@ func (throttler *Throttler) collectMySQLMetrics(ctx context.Context) error {
 
 // refreshMySQLInventory will re-structure the inventory based on reading config settings
 func (throttler *Throttler) refreshMySQLInventory(ctx context.Context) error {
-	log.Infof("refreshing MySQL inventory")
+	//log.Infof("refreshing MySQL inventory")
```

Comment:
@shlomi-noach are we ok to remove these Info logs?

Reply:
I had  commented these out because they were spamming the logs. @shlomi-noach: I can uncomment (or delete ...) , let me know.

Reply:
fixed